pacstall/__main__.py

```diff
@@ -41,16 +40,15 @@
         show_locals=True
     )  # --> Install Rich's traceback handler for better looking tracebackes
 
-    if getuser() == "root":
-        message.fancy("error", "Pacstall cannot be run as root")
-        sys.exit(1)
-
     args = parse_arguments()
     print(args)
 
+    if getuser() != "root":
+        message.fancy("error", "Pacstall needs to be launched as root!")
+        sys.exit(64)  # --> command line usage error
```

Comment:
If we're going to be using a set of standard error codes, we should either stick to the Linux Standard or create our own and reference them by a common enum

According to [this article](https://www.baeldung.com/linux/status-codes) code `126` would be better.

Reply:
IMO we should have our own values hardcoded in some kind of enum and use that

Reply:
> If we're going to be using a set of standard error codes, we should either stick to the Linux Standard or create our own and reference them by a common enum
> 
> According to [this article](https://www.baeldung.com/linux/status-codes) code `126` would be better.

I got `64` from the [`sysexits.h`](https://www.freebsd.org/cgi/man.cgi?query=sysexits) (this is a standard UNIX header). From what I can see that article (it's exit codes) is for bash scripts (https://stackoverflow.com/a/1101969).

Reply:
> IMO we should have our own values hardcoded in some kind of enum and use that

Yeah, we could use that for specific error codes, but I think for common errors like this one we should follow the standard error codes.

Do you want me to add an Enum of all the error codes (the standard ones for now) in `pacstall/api/error_codes.py`?

Reply:
Yup, that sounds good

Reply:
@paulcosma97 done